action.yml

```diff
@@ -43,12 +40,8 @@ runs:
         snap = launchpad.snaps.getByName(name="${{inputs.edgex_snap}}", owner=edgex_team)
 
         # it's not obvious from the documentation - but we need to specify the channel used for snapcraft, otherwise we get an old version
-        if bool ("${{inputs.build1}}"):
-          amd64 = release.getDistroArchSeries(archtag="${{inputs.build1}}")
-          request = snap.requestBuild(archive=ubuntu.main_archive, distro_arch_series=amd64, pocket='Updates', channels={"snapcraft":"edge"})
-          print(request)
-        if bool ("${{inputs.build2}}"):
-          arm64 = release.getDistroArchSeries(archtag="${{inputs.build2}}")
-          request = snap.requestBuild(archive=ubuntu.main_archive, distro_arch_series=arm64, pocket='Updates', channels={"snapcraft":"edge"})
+        if bool ("${{inputs.architecture}}"):
```

Comment:
Maybe the if statement isn't needed. This is a required input. Isn't it always there?